apps/transport/lib/transport/gtfs_data.ex

```diff
@@ -79,48 +107,83 @@ defmodule Transport.GTFSData do
     12 => {0.00171661376953125, 0.0011294445140095472}
   }
 
-  def create_gtfs_stops_materialized_view(zoom_level)
-      when is_integer(zoom_level) and zoom_level in 1..12 do
-    {:ok, %{rows: [[count]], num_rows: 1}} =
+  def list_views(pattern) do
```

Comment:
Peux-tu documenter les valeurs indiquées dans `@zoom_levels` juste au-dessus ?

Reply:
Je ferai ça oui

Reply:
@AntoineAugusti j'ai refait le processus de génération (tout en JS pour faire simple au niveau de l'explication) et j'ai noté tout ça dans a220a9444244eec0a40203d8da454116f821ee27. Dis-moi si ça te paraît suffisamment clair !